src/Tribe/Cache/Cache_Interface.php

```diff
@@ -18,18 +18,22 @@ public function reset_all();
 	 *
 	 * Please note that the list is aware of supported types.
 	 *
+	 * @param array $post_types An array of post types overriding the supported ones.
+	 *
 	 * @return array
 	 */
-	public function posts_without_ticket_types();
+	public function posts_without_ticket_types( array $post_types = null );
```

Comment:
I'm always a little nervous about changes to interface and abstract definitions: are we sure there's no risk of a dependent plugin like ET+ using this interface (since if it is left behind - even temporarily - during the next update process, fatals could result) or at least are we sure the necessary safeguards are in place?